.circleci/config.yml

```
@@ -9,8 +9,8 @@ commands:
           echo 'export ENZOTEST_DIR=$HOME/enzo_test' >> $BASH_ENV
           echo 'export ANSWER_NAME="push_suite"' >> $BASH_ENV
           echo 'export HYPRE=hypre-2.11.2' >> $BASH_ENV
-          # tag the tip so we can go back to it
-          git fetch --tags origin
+          # get tags from the main repository (for the current gold standard)
+          git fetch --tags https://github.com/enzo-project/enzo-dev
           git tag tip
```

We should move the original comment down to the line below.
```suggestion
          # tag the tip so we can go back to it
          git tag tip
```